lib/opFns.js

```diff
@@ -829,6 +830,7 @@ function subMemUsage (runState, offset, length) {
  * a string is instead returned. The function also subtracts the amount of
  * gas need for memory expansion.
  * @method memLoad
+ * @param runState
  * @param {BN} offset where to start reading from
  * @param {BN} length how far to read
  * @return {Buffer|String}
```

Comment:
Is it ``return`` or ``returns``?

Reply:
Either is valid, I believe `@returns` is more prevalent in the codebase from what I see currently